lsp-protocol.el

```diff
@@ -624,6 +626,8 @@ See `-let' for a description of the destructuring mechanism."
  (CompletionItemKindCapabilities nil (:valueSet))
  (CompletionItemTagSupportCapabilities (:valueSet) nil)
  (CompletionOptions nil (:resolveProvider :triggerCharacters :allCommitCharacters))
+ (InlineCompletionItem (:insertText) (:filterText :range :command))
```

Comment:
Please sort this

Reply:
I've added the relevante interfaces to the end of the list with the version of the protocol that supports them.

Let me know if you want to have the whole interface definition sorted.